transport/x/cherami/example/client.go

```diff
@@ -0,0 +1,91 @@
+// Copyright (c) 2016 Uber Technologies, Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+package example
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"go.uber.org/yarpc"
+	"go.uber.org/yarpc/transport/x/cherami"
+	"go.uber.org/yarpc/transport/x/cherami/example/thrift/example/exampleserviceclient"
+
+	"github.com/stretchr/testify/assert"
+	cherami_client "github.com/uber/cherami-client-go/client/cherami"
+)
+
+// TestCheramiYARPC will create a yarpc client (using cherami transport), and issue an one way rpc call to the server
+// To run this example, the cherami server(https://github.com/uber/cherami-server) needs to be started locally first
+// And the destination(/test/dest) and consumer group(/test/dest_cg) needs to be created using cherami-cli
+func TestCheramiYARPC(t *testing.T) {
```

Comment:
It used to be an example but with crossdocker this can modified to be an example. The file name is intentional. The test needs to be run with cherami server running, so we can't enable it. We'll move this to an integration test using crossdocker.